### DISCUSSION TITLE
Drop 2 samples - or n samples?

### DISCUSSION THREAD

**Discussion by Fuechsl** [voidtrance/voron-klipper-extensions#6] [Ideas]
Hi there,

First: Thanks a lot for this great extensions, I really like the drop sample feature! That's where I have a question: If I wanted to drop 2 samples instead of 1, could I just duplicate line 59 to 61 in the settling_probe.py? Would it make sense to have a parameter for this number in your script, so I could also drop 2 or 3 samples, if needed?

Regards, Martin

**Comment by voidtrance** [comment]
> Hi there,
> 
> First: Thanks a lot for this great extensions, I really like the drop sample feature! That's where I have a question: If I wanted to drop 2 samples instead of 1, could I just duplicate line 59 to 61 in the settling_probe.py? Would it make sense to have a parameter for this number in your script, so I could also drop 2 or 3 samples, if needed?
> 
> Regards, Martin

I think adding a parameter shouldn't be that difficult. I have been a bit busy with work lately but go ahead a file a feature request and I'll try to get to it in the next week or two.
